runtime/lua/vim/filetype.lua

```diff
@@ -1,3 +1,4 @@
+---@diagnostic disable: unused-local
```

Comment:
Probably better to move this to just before the `extensions` table (to keep it together with the `luacheck` directive).